### ISSUE TITLE
zig.nanorc issue

### ISSUE THREAD

**Issue by ahmadsiddeeq0** [scopatz/nanorc#335]
Error in /usr/share/nano/zig.nanorc on line 2: Color "latte" not understood
Error in /usr/share/nano/zig.nanorc on line 9: Command "tabgives" not understood

**Comment by scopatz** [comment]
PRs welcome!

**Comment by fcastilloec** [comment]
I updated to the latest version of `nano` (v5.3) from this page https://packages.debian.org/testing/editors/nano It's for a Debian based distro and you'll have to use a different package (try https://www.nano-editor.org/download.php)
The latest version doesn't show the error anymore. So it seems the code it's not at fault, but this project doesn't specify a minimum version of nano that it's compatible with. Perhaps adding something to the README might be a great idea.

**Comment by pedroarmindo1** [comment]
To fix this, go to /usr/share/nano/ and edit the file zig.nanorc
`nano zig.nanorc`
On file put // in tabgives

https://prnt.sc/wlv3mb

**Comment by ghost** [comment]
just delete the zig file

**Comment by rasa** [comment]
@scopatz said:
> PRs welcome!

I locally created two branches [`pre-5.0`](https://github.com/rasa/nanorc/tree/pre-5.0) and [`pre-4.5`](https://github.com/rasa/nanorc/tree/pre-4.5), that quiets nano's:
```
Error in /home/twig/.nano/zig.nanorc on line 2: Color "latte" not understood
Error in /home/twig/.nano/zig.nanorc on line 9: Command "tabgives" not understood
Error in /home/twig/.nano/rego.nanorc on line 15: Color "purple" not understood
```
But I can only submit a PR that gets merged into master. They really should be merged into `pre-5.0` and `pre-4.5` branches.

I will submit to master, and perhaps you can create the branches on your end. Thanks.
